### PR TITLE
Image and Input

### PR DESCRIPTION
**Pull request by Chiragmodi01**
image
image

### CODE REVIEW

File: Docsite/Docsite Comp CSS/Input.css [hunk 1]
```diff
@@ -0,0 +1,470 @@
...
+    background-color: rgba(51, 204, 204, 0.6);
+}
+.input-secondary:focus {
+    box-shadow: 1px 1px 6px 0 var(--primary-dark);
+    border-color: transparent;
+}
+.input-dark {
+    color: var(--primary-bold);
+    border-color: var(--primary-bold);
+    background-color: rgba(51, 153, 153, 0.6);
```
Comment (teja538) [inline Docsite/Docsite Comp CSS/Input.css:86]: use variables to define colors and use either rgb or hex

File: Docsite/Docsite Comp CSS/Input.css [hunk 2]
```diff
@@ -0,0 +1,470 @@
...
+}
+.input-dark:focus {
+    box-shadow: 1px 1px 6px 0 var(--primary-bold);
+    border-color: transparent;
+}
+.input-lightgrey {
+    color: #818182;
+    border-color: #818182;
+    background-color: #F1F1F1;
```
Comment (teja538) [inline Docsite/Docsite Comp CSS/Input.css:95]: here also use variables for colors

File: Docsite/Docsite Comp CSS/Input.css [hunk 3]
```diff
@@ -0,0 +1,470 @@
...
+
+/* input-box-wrapper */
+.comp-display-wrapper > .input-box-wrapper {
+    margin: auto;
+    margin-bottom: 1.5rem;
+}
+.comp-display-wrapper > .underline {
+    margin: 0  0 1.5rem 5rem;
+}
+
```
Comment (teja538) [inline Docsite/Docsite Comp CSS/Input.css:11]: remove extra spaces in css files

**Review by teja538 — COMMENTED** [review]
in input.css file at most of the places you did not used variables for colors. Use variables for colors. UI and rest of the code looks good

File: Docsite/Image.html [hunk 4]
```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./Docsite Comps CSS/Image.css">
```
Comment (agrawalshanu708) [inline Docsite/Image.html:7]: add meta tags of author, description

File: Docsite/Image.html [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+            You can also customize your solid input. Add any following class to your input-container to alter textbox's size - <span class="class-name">sm, md, lg</span> and we also have 8 different types of colors in textbox - <span class="class-name">primary, secondary, dark, lightgrey, success, warning, error and info</span>. And the default one is in basic white color. To use these colors, Just add the class <span class="class-name">input</span> and add any of the color mentioned above, following by an hifen(-) before.
+        <br>
+        Keep in mind that the <span class="class-name">for</span> attribute of label element, should match to the id of input element. Both sould be unique to another set of input textbox, if you're using multiple.</p>
+        <div class="comp-display-wrapper flex flex-row wrap gap-3">
+            <div class="image-container round-img-container">
+                <img src="https://picsum.photos/id/237/250/250" alt="img" class="img-responsive">
+            </div>
+            <div class="image-container square-img-container">
+                <img src="https://picsum.photos/id/237/250/250" alt="img" class="img-responsive">
+            </div>
```
Comment (agrawalshanu708) [inline Docsite/Image.html:65]: try to download all images in one folder and attach link from it

**Review by agrawalshanu708 — APPROVED** [review]

**Pull request merged**